spec/controllers/external_authors_controller_spec.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ExternalAuthorsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { FactoryGirl.create(:user) }
+
+  describe "GET #get_external_author_from_invitation" do
+    it "needs a valid invitaiton" do
+      get :get_external_author_from_invitation, invitation_token: "None existent"
+      it_redirects_to_with_error(root_path, "You need an invitation to do that.")
+    end
+   
+    it "needs to have a story assocated with the invitation" do
+      invitation = FactoryGirl.create(:invitation)
+      get :get_external_author_from_invitation, invitation_token: invitation.token
+      it_redirects_to_with_error(signup_path(invitation.token), "There are no stories to claim on this invitation. Did you want to sign up instead?")
+    end
+  end
+
+  describe "GET #complete_claim" do 
+    it "should give the user the works" do
+      external_author = FactoryGirl.create(:external_author)
+      user = FactoryGirl.create(:user)
```

Comment:
You shouldn't need this since you have `let(:user) { FactoryGirl.create(:user) }` at the top.